packages/utilities/psammead-assets/src/svgs.jsx

```diff
@@ -0,0 +1,173 @@
+      />
+      <path
+        stroke="#FFF"
+        strokeWidth=".335"
+        d="M222.797 3.304l1.047 2.277h-.85l-1.745-2.277z"
+      />
+      <path fill="#FFF" d="M246.203 3.304l-1.714 2.277h-.85l1.024-2.277h1.54" />
+      <path
+        stroke="#FFF"
+        strokeWidth=".335"
+        d="M246.203 3.304l-1.714 2.277h-.85l1.024-2.277z"
+      />
+    </g>
+  ),
+  ratio: 10.4167,
+};
+
+const Svg = styled.svg`
+  display: block;
+  height: ${props => props.height / 16} rem;
+  width: ${props => props.width / 16}rem;
+  margin-top: 1.25rem;
+  fill: #fff;
+  @media screen and (-ms-high-contrast: active), print {
+    fill: windowText;
+  }
+`;
+
+export const getSVG = ({ group, ratio, height }) => {
```

Comment:
this can go in `psammead-assets/src/svgs.stories.jsx` as we dont need it for production